static/golang/discovery.html

```diff
@@ -0,0 +1,4 @@
+<html><head>
+    <meta name="go-import" content="knative.dev/discovery git https://github.com/knative-sandbox/discovery">
+    <meta name="go-source" content="knative.dev/discovery     https://github.com/knative-sandbox/discovery https://github.com/knative-sandbox/discovery/tree/master{/dir} https://github.com/knative-sandbox/discovery/blob/master{/dir}/{file}#L{line}">
```

Comment:
@grantr @evankanderson @mattmoor Is this by intention that the import namespace is `knative.de` for both, knative proper and `knative-sandbox` ? Shouldn't there be als a distinction on the import-path level as people can get confused that discovery belongs to knative itself when seeing the import path ? (e.g. I think it really would be cool to keep 1:1 mappings on all levels)

Reply:
Yes, and I think it was sort of a toss up wrt the discussion.  On the one hand, everything you say.  On the other, a flat namespace allows us greater ability to move things in/out of "core" (e.g. all of the `net-{foo}` stuff).

cc @markusthoemmes @tcnghia @pmorie

Reply:
yeah, IIRC there was some discussion around "stuff that'll likely make core". We should pick this up again and clarify in the respective doc I believe?

Reply:
yes a clarification would be very helpful, especially because this PR is referenced as a blueprint from "new-repo" issue template in the community repo. Maybe also adapt the template to maybe even remove this step from the template ?